src/wasm-binary.h

```diff
@@ -261,17 +261,32 @@ enum ASTNodes {
   F64Le = 0x9a,
   F64Gt = 0x9b,
   F64Ge = 0x9c,
```

Comment:
I wanted to follow the groups spec and the wast symbol names, not G.s names in their document. I expect the groups encoding document will use the groups operator naming. It's a burden to be reading one symbol from the wast and have opcodes named differently in the source code particularly with all the combinations of pre-fixes and suffixes.